src/utils/multer.js

```diff
@@ -0,0 +1,14 @@
+const multer = require("multer");
+const path = require("path");
+
+module.exports = multer({
+    storage: multer.diskStorage({}),
+    fileFilter: (req,file,cb)=>{
+        let ext = path.extname(file.originalname);
+        if(ext!== ".jpg" && ext !==".jpeg" ){
+            cb(new Error("Unsupported file type!"), false);
```

Comment:
Lets create our own error line `UnsupportedFileError` it will be more specific and will help in debugging instead of generic error